App deployed to apk branch does not run on my device.

**Describe the bug**

The apk generated by Travis CI isn't working on an Android device, but when I run the app directly from Android Studio (with the latest commit) it runs well.

Log from the apk generated by Travis CI:
```
java.lang.RuntimeException: Unable to start activity ComponentInfo{org.systers.mentorship/org.systers.mentorship.view.activities.SplashScreenActivity}: android.view.InflateException: Binary XML file line #2: Binary XML file line #2: Error inflating class androidx.constraintlayout.widget.ConstraintLayout
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3297)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3405)
        at android.app.ActivityThread.-wrap12(Unknown Source:0)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1994)
        at android.os.Handler.dispatchMessage(Handler.java:108)
        at android.os.Looper.loop(Looper.java:166)
        at android.app.ActivityThread.main(ActivityThread.java:7523)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:245)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:921)
     Caused by: android.view.InflateException: Binary XML file line #2: Binary XML file line #2: Error inflating class androidx.constraintlayout.widget.ConstraintLayout
     Caused by: android.view.InflateException: Binary XML file line #2: Error inflating class androidx.constraintlayout.widget.ConstraintLayout
     Caused by: java.lang.reflect.InvocationTargetException
```
(...)

**To Reproduce**
Steps to reproduce the behavior:
1. Go to 'https://github.com/systers/mentorship-android/blob/apk/app-debug.apk'
2. Install the apk on your device
3. Click on the App icon
4. See it open and close

**Expected behavior**

The application running normally.

**Smartphone (please complete the following information):**
 - Device: Huawei ...
 - OS: Android
 - Version: 8.0

**Additional context**

I'm thinking if it is because of this line on Travis CI: https://github.com/systers/mentorship-android/blob/develop/.travis.yml#L10 that is explicitly using `android-27` when AndroidX requires:
```
(...) compile SDK to Android 9.0 (API level 28) (...)
```
According to https://developer.android.com/jetpack/androidx

This happened after merging PR #178

--- a/.travis.yml
+++ b/.travis.yml
@@ -7,7 +7,7 @@ android:
     - build-tools-28.0.3
     - android-28
 before_install:
- - yes | ${ANDROID_HOME}/tools/bin/sdkmanager "platforms;android-27"
+ - yes | ${ANDROID_HOME}/tools/bin/sdkmanager "platforms;android-28"
 before_script:
  - chmod +x gradlew
  - chmod +x generate-apks.sh

--- a/buildSrc/src/main/java/Dependencies.kt
+++ b/buildSrc/src/main/java/Dependencies.kt
@@ -12,7 +12,7 @@ object Versions {
     const val kotlinVersion = "1.3.21"
     const val supportLib = "1.0.0"
     const val designSupportLib = "1.0.0"
-    const val constraintLayout = "1.1.2"
+    const val constraintLayout = "1.1.3"
     const val junit = "4.12"
     const val testRunner = "1.1.0"
     const val espresso = "3.1.0"

